home.html

```diff
@@ -0,0 +1,76 @@
+        <p>Trabajando durante más de 10 años para ti, siendo pioneros en la industria online de distribución de películas, series y shows de televisión de alta calidad.</p>
+        <input id="sb1" type="submit" value="Empezar a Ver"/>
+    </section>
+<!--Sección 2 de la página principal "HOME"-->
+    <section id="seccion2">
+        <img id="logoS2"src="imagenes/is2.png.png" alt="Logo dos"/>       
+    </section>   
+<!--Sección 3 de la página principal "HOME"-->
+    <section id="s3">
+        <h1 id="h1s3">Ofrecemos una gama completa de<br/>carácteristicas para un mejor servicio online</h1>
+    <section class="seccion3">
+        <img class="is3" src="imagenes/s3pagos_adobespark.png" alt="Logo Métodos de Pago"/>
+        <h3>Métodos de Pago</h3>
+        <p>Contamos con varios métodos de pago para que en la comodidad del lugar en el que te encuentres puedas disfrutar de la compra rápida de tus películas favoritas: transferencia, bitcoins, tarjetas de crédito y débito.</p>
+    </section>
+    <section class="seccion3">
+        <img class="is3" src="imagenes/s3protection_adobespark.png" alt="Logo Protección de Datos"/>
+        <h3>Protección de Datos</h3>
+        <p>Para tu seguridad contamos con un sistema completo para la protección de datos, por lo cual puedes estar seguro de que resguardaremos tu información de manera efectiva.</p>
+    </section>
+    <section class="seccion3">
+        <img class="is3" src="imagenes/invoice_adobespark.png" alt="Logo Facturación"/>
+        <h3>Generador de Factura</h3>
+        <p>Contamos con un sistema innovador que te permitirá generar tu factura ingresando tus datos de facturación y enviarla automáticamente mediante correo electrónico.</p>
+    </section>
+    <section class="seccion3">
+        <img class="is3" src="imagenes/responder_adobespark.png" alt="Logo Respuesta Automática"/>
+        <h3>Respuesta Automática</h3>
+        <p>Te ofrecemos una respuesta automática para tus consultas más frecuentes y de igual manera un chat para una atención personalizada.</p>
+    </section>
+    <section class="seccion3">
+        <img class="is3" src="imagenes/plugin_adobespark.png" alt="Logo Plugins Sociales"/>
+        <h3>Plugins Sociales</h3>
+        <p>Contamos con la integración de plugins para las diferentes redes sociales, esto te permitirá redirigirte a nuestras páginas oficiales y así enterarte de lo más nuevo, promociones y mucho más.</p>
+    </section>
+    <section class="seccion3">
+        <img class="is3" src="imagenes/backup_adobespark.png" alt="Logo Backup"/>
+        <h3>Respaldo de Información</h3>
+        <p>Hemos trabajado durante 10 años en el respaldo autómatico para tu información y las películas o series que has adquirido; de esta manera no perderas tus compras o suscripciones.</p>
+    </section>
+</section>    
+<!--Footer-->
+<section>
+    <footer class="foot">
+        <h4> &COPY; CineOnline - Todos los derechos reservados</h4>
+    </footer>
+</section>

```

Comment:
La etiqueta `<section>` se puede omitir en este caso.
```suggestion
<footer class="foot">
    <h4> &COPY; CineOnline - Todos los derechos reservados</h4>
</footer>
```